vendor/assets/stylesheets/rolodex/settings/variables/_typography.scss

```diff
@@ -23,67 +23,12 @@ $font-family-open-sans: "Open Sans", Verdana, Helvetica, sans-serif;
 $font-family-helvetica: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
 $font-family-monospace: "Source Code Pro", Inconsolata, "Lucida Console", Terminal, "Courier New", Courier;
 
-// Weights
-
-$font-weight-book:   300;
```

Comment:
I pulled them as the variables didn't feel like they added much value or consistency, and are much longer to type out. 

It'll take some diligency to make sure we use `300`, `400`, and `600` accordingly but I think that'll become second nature fairly quickly.